docs/README.md

```diff
@@ -0,0 +1,7 @@
+# Docs
```

Comment:
Hmm, do we need currently the `docs` directory? I think we can remove it for now.